### DISCUSSION TITLE
How to integrate infolinks or revenuehits ads

### DISCUSSION THREAD

**Discussion by 92sajid110** [vercel/next.js#15515] [Help]
Hello i move from react to next js and i love nextjs because of static site generation and ssr.
My problem is i want to integrate ads from any provider rather than adsense in my website at certain areas.
So for adsense there is plugin but what for others any one have idea please explain me

**Comment by 92sajid110** [comment]
Any help please

**Reply by umdasali** [reply]
@92sajid110  hey have find the answer? if you found the answer please let me know. I'm looking for the same answer.

**Reply by 92sajid110** [reply]
No

**Comment by umdasali** [comment]
infolinks keep rejecting my ads approval because my site is not compatible with infolinks.

**Comment by 92sajid110** [comment]
Yes but i didn't find any way to do it

**Comment by AliWayne101** [comment]
anyone found the solution yet?
